tests/test_serialization.nim

```diff
@@ -149,3 +149,8 @@ suite "toJson tests":
     check:
        $original == $decoded
 
+  test "openArray[char]":
+    check:
+      "abc" == Json.decode(Json.encode(['a', 'b', 'c']), string)
```

Comment:
This check doesn't look right btw. Inside the Nim string, I expect to see escaped quotes.

Reply:
not sure what you mean - it decodes to a string, so it removes the quotes in the process - if you do `for c in ['a', 'b', 'c']` the intent is that the same thing happens as in `for c in "abc"`

Reply:
Ah, right, sorry. I didn't mention this is a roundtrip test.